lib/pdk/cli/convert.rb

```diff
@@ -22,12 +23,19 @@ module PDK::CLI
         log_level:           :info,
       )
 
-      PDK::CLI::Util.validate_template_opts(opts)
-
       if opts[:noop] && opts[:force]
         raise PDK::CLI::ExitWithError, _('You can not specify --noop and --force when converting a module')
       end
 
+      if opts[:'default-template']
+        raise PDK::CLI::ExitWithError, _('You can not specify --template-url and --default-template.') if opts[:'template-url']
+
+        opts[:'template-url'] = PDK::Util::TemplateURI.default_template_addressable_uri.to_s
```

Comment:
Should this be resetting template-ref as well?

Reply:
I thought that being able to do something like `pdk convert --default-template --template-ref 1.14.0` might be useful, so I left --template-ref alone. In terms of the saved ref in the answer file, it's stored as the fragment of the template-url uri, so it gets reset automatically for free.

Reply:
Ahhh ok 👍